receiver/jmxreceiver/receiver.go

```diff
@@ -119,6 +119,9 @@ func (jmx *jmxMetricReceiver) Start(ctx context.Context, host component.Host) er
 }
 
 func (jmx *jmxMetricReceiver) Shutdown(ctx context.Context) error {
+	if jmx.subprocess == nil {
+		return nil
+	}
```

Comment:
Sounds like an unrelated chance? If so, can you please move it to a separate PR?

Reply:
Will do.

Reply:
It's not unrelated actually. The receivers_test.go started to fail when I moved Validate out of the New function, because the receiver would be created but fail on shutdown.

I have peeled the change off to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/17554 anyway.